test_scripts/Defects/5_0/1387_Packet_validation_failed_if_send_PutFile_request_from_app_that_uses_v_2_protocol_and_app_that_uses_v_3_protocol_also_registered.lua

```diff
@@ -0,0 +1,93 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1387
+--
+-- Description:
+--
+-- Precondition:
+-- 1) "MaximumPayloadSize" = 1488 in smartDeviceLink.ini file
+-- 2) SDL and HMI are started
+-- 3) app1 is registered over v.2 protocol, app2 is registered over v.3 protocol
+-- 4) app1 is activated
+-- In case:
+-- 1) Send PutFile from app1 png_20kb.png.
+-- Expected result:
+-- 1) Request processed successfully
+--    SDL respond with success resultcode
+--    Note: There is no defect if only app1 (over v.2 protocol) registered
+-- Actual result:
+-- Packet validation failed
+-- SDL does't send any response to the app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local test = require("user_modules/dummy_connecttest")
+local mobile_session = require('mobile_session')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local count_of_requests = 1488
```

Comment:
please rename constant to `MaxPayloadSize`, this is not the count of the requests